indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +46,27 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
+        self, 
+        origin_dir,
+        destination_dir,
+        accepted_types: Tuple[str],
+        copy_files: bool = False,
+    ):
+        self.get_file_paths_from_dir(origin_dir, accepted_types, True)
+        if os.path.isdir(destination_dir):
+            if destination_dir[-1] != '/':
```

Comment:
most of this logic is handled by pathlib: https://realpython.com/python-pathlib/